service/src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java

```diff
@@ -277,46 +280,53 @@ void createAndDeleteWorkspace() {
-    WorkspaceRequest request =
-        defaultRequestBuilder(UUID.randomUUID())
-            .workspaceStage(WorkspaceStage.MC_WORKSPACE)
-            .build();
-    workspaceService.createWorkspace(request, USER_REQUEST);
-
-    Map<String, StepStatus> doFailures = new HashMap<>();
-    doFailures.put(DeleteGcpProjectStep.class.getName(), StepStatus.STEP_RESULT_FAILURE_RETRY);
-    doFailures.put(DeleteWorkspaceAuthzStep.class.getName(), StepStatus.STEP_RESULT_FAILURE_RETRY);
-    doFailures.put(DeleteWorkspaceStateStep.class.getName(), StepStatus.STEP_RESULT_FAILURE_RETRY);
-    FlightDebugInfo debugInfo = FlightDebugInfo.newBuilder().doStepFailures(doFailures).build();
+  void createMcWorkspaceDoSteps() {
+    WorkspaceRequest request = defaultRequestBuilder(UUID.randomUUID()).build();
+    Map<String, StepStatus> retrySteps = new HashMap<>();
+    retrySteps.put(CreateWorkspaceAuthzStep.class.getName(), StepStatus.STEP_RESULT_FAILURE_RETRY);
+    retrySteps.put(CreateWorkspaceStep.class.getName(), StepStatus.STEP_RESULT_FAILURE_RETRY);
+    FlightDebugInfo debugInfo = FlightDebugInfo.newBuilder().doStepFailures(retrySteps).build();
     jobService.setFlightDebugInfoForTest(debugInfo);
 
-    workspaceService.deleteWorkspace(request.workspaceId(), USER_REQUEST);
-    assertThrows(
-        WorkspaceNotFoundException.class,
-        () -> workspaceService.getWorkspace(request.workspaceId(), USER_REQUEST));
+    UUID createdId = workspaceService.createWorkspace(request, USER_REQUEST);
+    assertEquals(createdId, request.workspaceId());
   }
 
   @Test
-  void deleteWorkspaceUndoFailure() {
+  void createRawlsWorkspaceDoSteps() throws InterruptedException {
     WorkspaceRequest request =
         defaultRequestBuilder(UUID.randomUUID())
-            .workspaceStage(WorkspaceStage.MC_WORKSPACE)
+            .workspaceStage(WorkspaceStage.RAWLS_WORKSPACE)
             .build();
-    workspaceService.createWorkspace(request, USER_REQUEST);
+    // Ensure the auth check in CheckSamWorkspaceAuthzStep always succeeds.
+    doReturn(true).when(mockSamService).isAuthorized(any(), any(), any(), any());
+    Map<String, StepStatus> retrySteps = new HashMap<>();
+    retrySteps.put(
+        CheckSamWorkspaceAuthzStep.class.getName(), StepStatus.STEP_RESULT_FAILURE_RETRY);
+    retrySteps.put(CreateWorkspaceStep.class.getName(), StepStatus.STEP_RESULT_FAILURE_RETRY);
+    FlightDebugInfo debugInfo = FlightDebugInfo.newBuilder().doStepFailures(retrySteps).build();
+    jobService.setFlightDebugInfoForTest(debugInfo);
 
-    // Test "undo" flight with lastStepFailure. Because deletion cannot be undone, this workspace
-    // will still be deleted after "undoing" the flight.
-    FlightDebugInfo debugInfo = FlightDebugInfo.newBuilder().lastStepFailure(true).build();
+    UUID createdId = workspaceService.createWorkspace(request, USER_REQUEST);
+    assertEquals(createdId, request.workspaceId());
+  }
+
+  @Test
+  void createWorkspaceUndoSteps() {
```

Comment:
nit: is this for MC workspaces only? if so, can you include it in the method name like you did above?